packages/astro-embed-baseline-status/BaselineStatus.astro

```diff
@@ -0,0 +1,293 @@
+
+const title = BASELINE_DEFS[baseline].title;
+const baselineDate = getBaselineDate(feature);
+const description = getDescriptionDate(baseline, baselineDate);
+const year =
+	baseline === 'newly' && baselineDate ? baselineDate.split(' ')[1]! : '';
+const { chrome, edge, firefox, safari } = feature.browser_implementations || {};
+
+function getAriaLabel(
+	title: string,
+	year: string,
+	newlyAvailable: boolean,
+	chrome = 'no',
+	edge = 'no',
+	firefox = 'no',
+	safari = 'no'
+) {
+	if (title === 'Unknown availability') {
+		chrome = edge = firefox = safari = 'unknown';
+	}
+	return `Baseline: ${title}${year ? ` ${year}` : ''}${
+		newlyAvailable ? ` (newly available)` : ''
+	}. Supported in Chrome: ${
+		chrome === 'available' ? 'yes' : chrome
+	}. Supported in Edge: ${
+		edge === 'available' ? 'yes' : edge
+	}. Supported in Firefox: ${
+		firefox === 'available' ? 'yes' : firefox
+	}. Supported in Safari: ${safari === 'available' ? 'yes' : safari}.`;
+}
+---
+
+<baseline-status>
+	<h1>{feature.name}</h1>
+	<details>
+		<summary
+			aria-label={getAriaLabel(
+				title,
+				year,
+				baseline === 'newly',
+				chrome?.status,
+				edge?.status,
+				firefox?.status,
+				safari?.status
+			)}
+		>
+			<BaselineIcon support={baseline} />
+			<div class="baseline-status-title" aria-hidden="true">
+				<div>
+					{
+						!(baseline === 'limited' || baseline === 'no_data') && (
+							<strong>Baseline</strong>
+						)
+					}
+					{title}
+					{year}
+					{
+						baseline === 'newly' && (
+							<span class="baseline-badge">newly available</span>
+						)
+					}
+				</div>
+				<div class="baseline-status-browsers">
+					<span>
+						<BrowserIcon browser="chrome" />
+						<SupportIcon {baseline} browserImplementation={chrome} />
+					</span>
+					<span>
+						<BrowserIcon browser="edge" />
+						<SupportIcon {baseline} browserImplementation={edge} />
+					</span>
+					<span>
+						<BrowserIcon browser="firefox" />
+						<SupportIcon {baseline} browserImplementation={firefox} />
+					</span>
+					<span>
+						<BrowserIcon browser="safari" />
+						<SupportIcon {baseline} browserImplementation={safari} />
+					</span>
+				</div>
+			</div>
+			<span class="open-icon" aria-hidden="true">
+				<svg
+					xmlns="http://www.w3.org/2000/svg"
+					width="11"
+					height="7"
+					viewBox="0 0 11 7"
+					fill="none"
+				>
+					<path
+						d="M5.5 6.45356L0.25 1.20356L1.19063 0.262939L5.5 4.59419L9.80937 0.284814L10.75 1.22544L5.5 6.45356Z"
+						fill="currentColor"></path>
+				</svg>
+			</span>
+		</summary>
+		<p set:html={description} />
+		<p
+			set:html={baseline === 'no_data'
+				? ''
+				: `<a href="https://github.com/web-platform-dx/web-features/blob/main/features/${feature.feature_id}.yml" target="_top">Learn more</a>`}
+		/>
+	</details>
+</baseline-status>
+
+<style>
+	baseline-status {
+		--baseline-status-color-limited: light-dark(#ea8600, #f09418);
```

Comment:
`light-dark` seems a bit too new to rely on, no?

It also breaks if the website doesn't set `color-scheme` (though I guess that's an author error).

Reply:
It would be also nice to have some more custom properties for customization. In my implementation, I'm using a mix of shadow DOM parts and custom properties.